tests/server/routes/particularCohortRoute.test.js

```diff
@@ -0,0 +1,12 @@
+const request = require('supertest')
```

Comment:
Change the name of this file to reflect the path and filename of the route you are testing.